src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -110,6 +111,35 @@ const SubmissionDetails = _.flow(
   )(workflows)
 
   const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+
+  const archiveLimitYears = 6
+  const archiveLimitString = `${archiveLimitYears} year${archiveLimitYears > 1 ? 's' : ''}`
+
+  const archived = lastChangedDate => {
+    const dateYearsAgo = new Date(new Date().setFullYear(new Date().getFullYear() - archiveLimitYears))
```

Comment:
You should look in to https://date-fns.org/v2.21.3/docs/differenceInYears, we have this library already.

Reply:
You can find an example in `ServiceAlerts.js` of using `differenceInDays`, note that we import it from the `fp` submodule of date-fns